tests/trainers/test_segmentation.py

```diff
@@ -262,3 +263,23 @@ def test_freeze_decoder(
                 for param in model.model.segmentation_head.parameters()
             ]
         )
+
+    @pytest.mark.parametrize(
+        "class_weights", [torch.tensor([1, 2, 3]), np.array([1, 2, 3]), [1, 2, 3]]
```

Comment:
Can you also add a normal list here?

Reply:
The parameters already include a normal list, so covers PyTorch tensor, NumPy array and Python list.